docs/practitioner/introduction.md

```diff
@@ -32,3 +32,17 @@ The following certification covers 6 key areas that a green software practitione
 6. **Climate Commitments**: Understand the exact mechanism of reduction.
 
 Each of these chapters will introduce some new concepts and explain in detail why they are important in terms of the climate, and how you can apply them to your green software practices.
+
+## Principles, Patterns, and Practices.
+
+This course's key areas and content describe the **principles** of green software, a core set of competencies needed to define, run and build green software.
+
+A green software **pattern** is a specific example of how to apply one or more principles in a real-world example. Whereas principles describe the theory that underpins green software, patterns are the practical advice software practitioners can use in their software applications today. Patterns are vendor-neutral.
+
+Green software **practices** are patterns applied to a specific vendor's product and inform practitioners how to use that product in the most sustainable way possible. 
```

Comment:
Should we also say "A green software pattern..." so it's consistent with line 40?

Reply:
Makes sense will adjust!